src/crons/CommitStrip.ts

```diff
@@ -3,27 +3,35 @@ import got from 'got';
 import { decode } from 'html-entities';
 
 import { Cron, findTextChannelByName } from '../framework';
+import { KeyValue } from '../database';
 
 export default new Cron({
   enabled: true,
   name: 'CommitStrip',
   description:
     'Vérifie toutes les 30 minutes si un nouveau CommitStrip est sorti et le poste dans #gif',
-  schedule: '*/30 * * * *',
+  schedule: '5,35 * * * *',
   async handle(context) {
-    const latestCommitStrip = await getRecentCommitStrip(context.date);
-    if (!latestCommitStrip) {
-      return;
-    }
-    context.logger.info(`Found a new CommitStrip (${latestCommitStrip.id})`);
+    const strip = await getLastCommitStrip();
+
+    // vérifie le strip trouvé avec la dernière entrée
+    const lastStrip = await KeyValue.get<number>('Last-Cron-CommitStrip');
+    const gameStoreIdentity = strip?.id ?? null;
+    if (lastStrip === gameStoreIdentity) return; // skip si identique
+
+    await KeyValue.set('Last-Cron-CommitStrip', gameStoreIdentity); // met à jour sinon
+
+    if (!strip) return; // skip si pas de strip
```

Comment:
On devrait peut-être mettre cette condition plus tôt. je ne sais pas si on veut mettre `null` dans le KV quand il n'y a pas de strip

Reply:
Ça me dérange pas de mettre des valeurs falsy dans la DB, je préfère qu'on puisse savoir en DB qu'est ce qui a été vu a la dernière itération.

Reply:
ok